contracts/StarEthRate.sol

```diff
@@ -0,0 +1,52 @@
+pragma solidity 0.4.25;
+
+import "./lib/Ownable.sol";
+
+contract StarEthRate is Ownable {
+    uint256 public decimalCorrectionFactor;
+    uint256 public starEthRate;
+
+    event decimalCorrectionFactorSet(uint256 decimalCorrectionFactor);
+    event StarEthRateSet(uint256 starEthRate);
+
+    constructor(
+        uint256 _decimalCorrectionFactor,
+        uint256 _initialStarEthRate
+    ) public {
+        require(
+            _decimalCorrectionFactor > 0,
+            'Please pass a decimalCorrectionFactor above 0!'
+        );
+        require(_initialStarEthRate > 0, 'Please pass a starEthRate above 0!');
+
+        decimalCorrectionFactor = _decimalCorrectionFactor;
+        starEthRate = _initialStarEthRate;
+    }
+
+    function setDecimalCorrectionFactor(
+        uint256 _newdecimalCorrectionFactor
```

Comment:
small aesthetic change: `_newdecimalCorrectionFactor` to `_newDecimalCorrectionFactor`